.clang-format

```diff
@@ -0,0 +1,21 @@
+---
```

Comment:
Does .clang-format makes sense in a go SDK?

Reply:
if you want it to format your protofiles, then yes.